packages/cli/src/config.ts

```diff
@@ -24,21 +24,22 @@ function isStringRecord(v: unknown): v is Record<string, string> {
   );
 }
 
-export const defaultConfig = {
-  endpoint: '/api/graphql',
-  enumsAsStrings: false,
+export const DUMMY_ENDPOINT = 'SPECIFY_ENDPOINT_OR_SCHEMA_FILE_PATH_HERE';
+
+export const defaultConfig: Required<GQlessConfig> = {
```

Comment:
this default config type (Required\<GQlessConfig\>) doesn't take in account the deep fields in the introspection config